pkg/diagnostics/component_monitoring.go

```diff
@@ -381,6 +404,32 @@ func (c *componentMetrics) CryptoInvoked(ctx context.Context, component, operati
 	}
 }
 
+// JobTriggeredSuccess records the metrics for a job execution success event.
+func (c *componentMetrics) JobTriggeredSuccess(ctx context.Context, operation string, success bool, elapsed float64) {
+	c.jobTriggered(ctx, operation, success, elapsed, c.jobSuccessCount)
+}
+
+// JobTriggeredFailure records the metrics for a job execution failure event.
+func (c *componentMetrics) JobTriggeredFailure(ctx context.Context, operation string, success bool, elapsed float64) {
+	c.jobTriggered(ctx, operation, success, elapsed, c.jobFailureCount)
+}
+
+func (c *componentMetrics) jobTriggered(ctx context.Context, operation string, success bool, elapsed float64, countMeasure *stats.Int64Measure) {
```

Comment:
does it make sense to remove the `success bool` here? If the metric name includes `failure` vs `success` anyways I dont see there being much value in having this as an additional tag